cmd/krel/cmd/ff_test.go

```diff
@@ -0,0 +1,218 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cmd
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+
+	"k8s.io/release/pkg/command"
+	"k8s.io/release/pkg/git"
+)
+
+const (
+	pseudoReleaseBranch = "release-2.0"
+	testCommitMessage   = "This is my test commit"
+)
+
+type sut struct {
+	repo        *git.Repo
+	baseDir     string
+	bareCopyDir string
+}
+
+func newSUT(t *testing.T) *sut {
```

Comment:
I wonder if we could abstract this functionality a bit as I think it would be useful in testing many of the other commands as well :)

Reply:
Probably makes sense for a follow-up!

Reply:
Hm, I also have https://github.com/kubernetes/release/pull/1068 in flight, so I guess if we merge this PR first I will refactor it there for a better re-usability.